1-Draft/RFCXXXX-ScriptBlockLogging.md

```diff
@@ -0,0 +1,280 @@
+---
+RFC: RFCXXXX
+Author: Travis Plunk
+Status: Draft
+SupersededBy:
+Version: 1.0
+Area: Engine
+Comments Due: 08/15/2019
+Plan to implement: Yes
+---
+
+# Title
+
+Remote ScriptBlock Logging
+
+## Description
+
+This is an addition to the existing ScriptBlock creation logging.
+The goal is to make it easier to configure to collect the logs centrally.
+Because many ScriptBlocks contain sensitive information,
+it is advised to encrypt the logs locally against attacker that might compromise the local machine.
```

Comment:
We're not doing the encrypted logging for this iteration, I think it's better to call out the remote aspects of the logging as a security measure

Reply:
It’s two lines down.

Reply:
This is part of calling this out.  Pointing out the disadvantages of logging locally